custom_components/tahoma/cover.py

```diff
@@ -47,6 +47,19 @@
 COMMAND_STOP_IDENTIFY = "stopIdentify"
 COMMAND_UP = "up"
 
+COMMANDS_STOP = [COMMAND_STOP_IDENTIFY, COMMAND_STOP, COMMAND_MY]
+COMMANDS_STOP_TILT = [COMMAND_STOP_IDENTIFY, COMMAND_STOP, COMMAND_MY]
```

Comment:
Do you think  later these 2 lists will be different? Can we use only one for the moment?

Reply:
Not 100% sure yet... For the devices with tilt position that I have tested, the stop command didn't work, while the stop identify did. However, with this PR the stop command works based on the cancel method, if the command was executed from Home Assistant.

Another idea I had was to pull the running executions via the API; when you call stop without any running executions. But that would be for a next PR.